.gitattributes

```diff
@@ -0,0 +1,7 @@
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.php_cs.dist export-ignore
+/.phpstan.neon export-ignore
+/.styleci.yml export-ignore
+/.travis.yml export-ignore
+/Tests export-ignore
```

Comment:
Sorry to come back here once more, but according to the documentation, the path should end with an slash (https://git-scm.com/book/en/v2/Customizing-Git-Git-Attributes)

Reply:
Obviously I was able to change your pull request on my own ^^ Will merge as soon as travis is passed

Reply:
This is only essential if there could be a toplevel file called `Tests` which is obviously not the case here. ;-)